src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -585,6 +585,7 @@ def load_arguments(self, _):
             c.argument('custom_data', help='Custom init script file or text (cloud-init, cloud-config, etc..)', completer=FilesCompleter(), type=file_type)
             c.argument('secrets', multi_ids_type, help='One or many Key Vault secrets as JSON strings or files via `@{path}` containing `[{ "sourceVault": { "id": "value" }, "vaultCertificates": [{ "certificateUrl": "value", "certificateStore": "cert store name (only on windows)"}] }]`', type=file_type, completer=FilesCompleter())
             c.argument('assign_identity', nargs='*', arg_group='Managed Service Identity', help="accept system or user assigned identities separated by spaces. Use '[system]' to refer system assigned identity, or a resource id to refer user assigned identity. Check out help for more examples")
+            c.argument('terminate_notification', help='Terminate Scheduled Event. Configurable length of time a Virtual Machine being deleted will have to potentially approve the Terminate Scheduled Event before the event is auto approved (timed out). The configuration must be specified in ISO 8601 format. e.g. PT5M')
```

Comment:
I suggest we reword the whole comment a bit. I know it is from the swagger, but it makes me feel a bit out of breath to read it :). Maybe `length of time a notification to be sent to the VM on the instance metadata server till the VM gets deleted. `? Also please cross check the `PT5M` is 8061 format?

Reply:
Please add api-version range

Reply:
> 
> 
> I suggest we reword the whole comment a bit. I know it is from the swagger, but it makes me feel a bit out of breath to read it :). Maybe `length of time a notification to be sent to the VM on the instance metadata server till the VM gets deleted. `? Also please cross check the `PT5M` is 8061 format?

Yes. It's hard to read. PT5M is an example provided by the swagger file.

Reply:
> 
> 
> Please add api-version range

OK